pkgs/os-specific/linux/atop/default.nix

```diff
@@ -9,31 +19,55 @@ stdenv.mkDerivation rec {
     sha256 = "nsLKOlcWkvfvqglfmaUQZDK8txzCLNbElZfvBIEFj3I=";
   };
 
-  buildInputs = [zlib ncurses];
+  nativeBuildInputs = if withAtopgpu then [ python3.pkgs.wrapPython ] else [ ];
+  buildInputs = [ zlib ncurses ] ++ (if withAtopgpu then [ python3 ] else [ ]);
+  pythonPath = if withAtopgpu then [ python3.pkgs.pynvml ] else [ ];
 
   makeFlags = [
-    "SCRPATH=$out/etc/atop"
-    "LOGPATH=/var/log/atop"
-    "INIPATH=$out/etc/rc.d/init.d"
-    "SYSDPATH=$out/lib/systemd/system"
-    "CRNPATH=$out/etc/cron.d"
-    "DEFPATH=$out/etc/default"
-    "ROTPATH=$out/etc/logrotate.d"
+    "DESTDIR=$(out)"
+    "BINPATH=/bin"
+    "SBINPATH=/bin"
+    "MAN1PATH=/share/man/man1"
+    "MAN5PATH=/share/man/man5"
+    "MAN8PATH=/share/man/man8"
+    "SYSDPATH=/lib/systemd/system"
+    "PMPATHD=/lib/systemd/system-sleep"
```

Comment:
Why do they not include $out?

Reply:
They are all used as like this: `$(DESTDIR)$(BINPATH)`, so `$out` is correctly prependet.

Reply:
@SuperSandro2000 Works for you? Alternativeley I can `DESTDIR=""` and fully qualify all the names, but then I'd have to go through the Makefile again because I only mentioned the keys that were relevant, and were wrong by default with `DESTDIR=$(out)`.